Life-Expectancy-and-GDP-Starter/life_expectancy_gdp.ipynb

```diff
@@ -141,6 +141,392 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Trend of GDP over time\n",
+    "plt.figure(figsize=(10, 6))\n",
+    "sn.lineplot(x=\"Year\", y=\"GDP\", data=gdp_data, marker='o', label='GDP')\n",
+    "plt.title('Trend of GDP over time')\n",
+    "plt.xlabel('Year')\n",
+    "plt.ylabel('GDP')\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The GDP trend also shows growth, although constant, there is a slight decrease in 2018 and then it continues to grow.\n",
```

Comment:
"The GDP trend also shows growth. The growth is generally constant, but there is a slight decrease in GDP from 2008 to 2009."

Separate the sentence into two sentences and fix the typo.